docs/src/test-annotations-js.md

```diff
@@ -159,7 +159,7 @@ You can also filter tests in the configuration file via [`property: TestConfig.g
 
 ## Annotate tests
 
-If you would like to annotate your tests with something more substantial than a tag, you can do that when declaring a test. Annotations have a `type` and a `description` for more context, and will be visible in the test report.
+If you would like to annotate your tests with something more substantial than a tag, you can do that when declaring a test. Annotations have a `type` and a `description` for more context. Annotations will be visible in the test report, except those where `type` starts with `_` symbol.
```

Comment:
Strictly speaking, everything is visible as a part of the test reporter API, so I would be more specific and mention that annotations that start with _ are not rendered by the built-in reporters.

Reply:
@pavelfeldman updated text. I've pointed that only built-in HTML reporter hides "_" annotations.